examples/simple/simple_test.go

```diff
@@ -24,19 +24,38 @@ import (
 	eventstore "github.com/looplab/eventhorizon/eventstore/memory"
 	readrepository "github.com/looplab/eventhorizon/readrepository/memory"
 
+	"testing"
+
 	"github.com/looplab/eventhorizon/examples/domain"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
-func Example() {
```

Comment:
The question is how to make the build stable. Right now you treat Example* as a test because it is not excluded from Makefile test task. However, `for i in {1..25}; do go test . ; done` on the package example/simple may occasionally fail. Turning example into test brings stability the entire `go test` launch.